plugins/user/cpubyuser

```diff
@@ -55,50 +67,48 @@ fi
 if [ "$1" = "config" ]; then
 	echo "graph_args --base 1000 -r --lower-limit 0"
 	echo "graph_title CPU usage, by user"
-	echo "graph_category cpu"
+	echo "graph_category system"
```

Comment:
I have the feeling that `cpu` is quite suitable here, since the plugin focuses on the CPU usage.

Reply:
Rolled back.